apps/cli/src/cli/utils.ts

```diff
@@ -159,7 +160,7 @@ export function clirun(
         useSession.log.debug(`\n\n${(error as Error)?.stack}\n\n`);
       }
       useSession.log.error((error as Error).message);
-      await logVersions(useSession, false);
+      logVersions(useSession, versions, false);
```

Comment:
logVersions is now not async, which was giving challenges in showing the error message in a different place than the process.exit when multiple things were running.